### PR TITLE
chore(scripts): update script generator and readme template

### PR DESCRIPTION
**Pull request by antonsamper**
#### Reference
- https://financialtimes.atlassian.net/browse/CPP-149

#### Notes
- Aligns README with [README Template](https://github.com/Financial-Times/next/wiki/README-template)
- I have updated the script to generate the documentation so that writing the documentation becomes easier
  - Happy to discuss this as a team  😄

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# n-heroku-tools (nht) [![CircleCI](https://circleci.com/gh/Financial-Times/n-heroku-tools.svg?style=svg&circle-token=33bcf2eb98fe2e875cc66de93d7e4a50369c952d)](https://circleci.com/gh/Financial-Times/n-heroku-tools)
```
Comment (apaleslimghost) [inline scripts/README.md:1]: maybe rename this file so it's clearer it's a template?

File: README.md [hunk 2]
```diff
@@ -1,21 +1,62 @@
...
+
+```sh
+npm install --save-dev @financial-times/n-heroku-tools 
 ```
 
 ### Usage
 
 Use `nht` or `n-heroku-tools` on the command line:
 
 ```
```
Comment (andygout) [inline README.md:58]: So everything in the README from here downwards has been created by the `make docs` script? Neat!

File: README.md [hunk 3]
```diff
@@ -1,21 +1,62 @@
-# n-heroku-tools (nht)
+# n-heroku-tools (nht) [![CircleCI](https://circleci.com/gh/Financial-Times/n-heroku-tools.svg?style=svg&circle-token=33bcf2eb98fe2e875cc66de93d7e4a50369c952d)](https://circleci.com/gh/Financial-Times/n-heroku-tools)
 
-This library is a command line tool that orchestrates [Heroku](https://www.heroku.com/) and [Amazon S3](https://aws.amazon.com/s3/) deployments for [Next](https://github.com/Financial-Times/next/wiki), based on configuration in the [Next service registry](https://next-registry.ft.com/v2/) and [Vault](https://www.vaultproject.io/).
+This library is a command line tool that orchestrates [Heroku](https://www.heroku.com/) and [Amazon S3](https://aws.amazon.com/s3/) deployments for [Next](https://github.com/Financial-Times/next/wiki), based on configurations in the [Next service registry](https://next-registry.ft.com/v2/) and [Vault](https://www.vaultproject.io/).
 
-### Installation
+🤖 **This file is generated automatically. If you need to make updates please modify `/scripts/README.md` and afterwards run `make docs`.**
```
Comment (andygout) [inline README.md:5]:
```suggestion
🤖 **This file is generated automatically. If you need to make updates please modify `/scripts/readme-template.md` and afterwards run `make docs`.**
```

I think the file name changed from `README.md` to `readme-template.md`.

File: scripts/readme-template.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# n-heroku-tools (nht) [![CircleCI](https://circleci.com/gh/Financial-Times/n-heroku-tools.svg?style=svg&circle-token=33bcf2eb98fe2e875cc66de93d7e4a50369c952d)](https://circleci.com/gh/Financial-Times/n-heroku-tools)
+
+This library is a command line tool that orchestrates [Heroku](https://www.heroku.com/) and [Amazon S3](https://aws.amazon.com/s3/) deployments for [Next](https://github.com/Financial-Times/next/wiki), based on configurations in the [Next service registry](https://next-registry.ft.com/v2/) and [Vault](https://www.vaultproject.io/).
```
Comment (andygout) [inline scripts/readme-template.md:3]: Are the Amazon S3 deployments for hashed assets and static websites only?

I.e. I am guessing they do not include S3-hosting for Serverless applications?

Comment (antonsamper) [reply scripts/readme-template.md:3]: correct! it simply uploads files to S3 but it wont run serverless applications.

**Pull request merged**